docs/index.rst

```diff
@@ -34,7 +34,7 @@ In addition, there are several extension packages which can be installed separat
     pip install opentelemetry-instrumentation-{instrumentation}
 
 These are for exporter and instrumentation packages respectively.
-Some packages can be found in :scm_web:`instrumentation <instrumentation/>` and :scm_web:`exporter <exporter/>`
+Some packages can be found in `instrumentation <https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation>`_ and :scm_web:`exporter <exporter/>`
```

Comment:
We actually shouldn't even have this part since all of the instrumentations exist in the contrib repo. As well, can you link the "exporter" to [this](https://github.com/open-telemetry/opentelemetry-python/tree/main/exporter) instead of the `:scm_web`? We should be consistent with the following sentence: 

`The remaining packages can be found at the 
`Contrib repo instrumentation <https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/instrumentation>`_
and `Contrib repo exporter <https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/exporter>`_ directories.`

Reply:
@marcosflobo do you want to modify this PR to remove this line instead?

Reply:
@ocelotl ok, I'll remove this line then